### PR TITLE
adding an additional check to see if filevault is enabled.

### PR DESCRIPTION
**Pull request by weswhet**
more detailed FileVault check to avoid false positives.

### CODE REVIEW

File: code/client/munkilib/munkicommon.py [hunk 1]
```diff
@@ -2772,15 +2772,33 @@ def blockingApplicationsRunning(pkginfoitem):
...
-    enabled as it may return true on a machine with
-    FileVault disabled."""
-    cmd = ['/usr/bin/fdesetup', 'supportsauthrestart']
-    if subprocess.check_output(cmd).strip() == 'true':
+    """Check if FileVault is enabled then checks
+    if an Authorized Restart is supported, returns True
+    or False accordingly.
+    """
+    display_debug1('Checking if FileVault is Enabled...')
+    active_cmd = ['/usr/bin/fdesetup','isactive']
```
Comment (gregneagle) [inline code/client/munkilib/munkicommon.py]: Need a space after the comma.

File: code/client/munkilib/munkicommon.py [hunk 2]
```diff
@@ -2772,15 +2772,33 @@ def blockingApplicationsRunning(pkginfoitem):
...
+        display_warning('FileVault appears to be Disabled...')
+        return False
+    display_debug1(
+        'Checking if FileVault can perform an Authorized Restart...')
+    support_cmd = ['/usr/bin/fdesetup', 'supportsauthrestart']
+    try:
+        is_supported = subprocess.check_output(support_cmd).strip()
+    except subprocess.CalledProcessError:
+        display_warning(
+            'FileVault appears to not support an Authorized Restart...')
```
Comment (gregneagle) [inline code/client/munkilib/munkicommon.py]: Or more accurately/specifically -- the current machine/hardware does not support authrestart.

File: code/client/munkilib/munkicommon.py [hunk 3]
```diff
@@ -2772,15 +2772,33 @@ def blockingApplicationsRunning(pkginfoitem):
...
+        return False
+    if is_active == 'true' and is_supported == 'true':
+        display_debug1(
+            'FileVault is on and Supports an AuthResrtart...')
```
Comment (gregneagle) [inline code/client/munkilib/munkicommon.py]: Spelling

**Comment by gregneagle** [comment]
@wes: could you address my notes?

**Comment by weswhet** [comment]
Sorry about the delay on this, i'm doing a bit of a rewrite on this. to better handle 10.6-10.12 machines.

**Comment by weswhet** [comment]
Ugh. Sorry should have squashed the commits for merging in the upstream branch.

**Comment by weswhet** [comment]
I'll fix the issues with the travis build.

**Comment by weswhet** [comment]
Moving changes to new branch.

**Pull request closed without merging**